infra/gcb/builds_status.py

```diff
@@ -1,10 +1,28 @@
+# Copyright 2020 Google Inc.
```

Comment:
let's revert most of the changes here, as you've reimplemented most of the functionality.